bezier_library/src/bezier_library.cpp

```diff
@@ -578,6 +578,103 @@ bool Bezier::keepUpperPartofDilatedMesh(vtkSmartPointer<vtkPolyData> &base_polyd
   return true;
 }
 
+bool Bezier::removeIsolatedTrianglesFilter(vtkSmartPointer<vtkPolyData> &polydata, const unsigned minimal_number_of_cells)
+{
+  //FIXME add Operation observer
+  vtkSmartPointer<vtkPolyDataConnectivityFilter> connectivityFilter =
+      vtkSmartPointer<vtkPolyDataConnectivityFilter>::New();
+  connectivityFilter->SetExtractionModeToAllRegions();
+  connectivityFilter->ColorRegionsOff();
+#if VTK_MAJOR_VERSION <= 5
```

Comment:
We do not support VTK versions below 7.0 so remove these pre-processor directives.